controllers/submariner/submariner_controller.go

```diff
@@ -416,6 +416,19 @@ func newEnginePodTemplate(cr *submopv1a1.Submariner) corev1.PodTemplateSpec {
 
 	// Create Pod
 	terminationGracePeriodSeconds := int64(1)
+
+	// Default healthCheck Values
+	healthCheckEnabled := true
+	// The values are in seconds
+	healthCheckInterval := uint64(1)
+	healthCheckMaxPacketLossCount := uint64(15)
```

Comment:
This defaults to 5 in submariner code. Perhaps we should not default here, ie omit or pass empty strings for the env vars if `HealthCheckSpec`  is not set and let submariner default them.

Reply:
Made the value to 5, I  could refactor the later this later if required.